documentation/source/overview/studies.rst

```diff
@@ -5,96 +5,96 @@ Studies using SCT
+-  Martin et al. `A Novel MRI Biomarker of Spinal Cord White Matter Injury: T2*-Weighted White Matter to Gray Matter Signal Intensity Ratio. <https://www.ncbi.nlm.nih.gov/pubmed/28428212>`__ **AJNR** 2017
+-  David et al. `The efficiency of retrospective artifact correction methods in improving the statistical power of between-group differences in spinal cord DTI. <http://www.sciencedirect.com/science/article/pii/S1053811917305220>`__ **Neuroimage** 2017
+-  Battiston et al. `An optimized framework for quantitative Magnetization Transfer imaging of the cervical spinal cord in vivo. <http://onlinelibrary.wiley.com/doi/10.1002/mrm.26909/full>`__ **Magnetic Resonance in Medicine** 2017
+-  Rasoanandrianina et al. `Region-specific impairment of the cervical spinal cord (SC) in amyotrophic lateral sclerosis: A preliminary study using SC templates and quantitative MRI (diffusion tensor imaging/inhomogeneous magnetization transfer). <http://onlinelibrary.wiley.com/doi/10.1002/nbm.3801/full>`__ **NMR Biomed** 2017
+-  Weber et al. `Thermal Stimulation Alters Cervical Spinal Cord Functional Connectivity in Humans. <http://www.sciencedirect.com/science/article/pii/S0306452217307637>`__  **Neurocience** 2017 
+-  Grabher et al. `Neurodegeneration in the Spinal Ventral Horn Prior to Motor Impairment in Cervical Spondylotic Myelopathy. <http://online.liebertpub.com/doi/abs/10.1089/neu.2017.4980>`__ **Journal of Neurotrauma** 2017
+-  Duval et al. `Scan–rescan of axcaliber, macromolecular tissue volume, and g-ratio in the spinal cord. <http://onlinelibrary.wiley.com/doi/10.1002/mrm.26945/full>`__ **Magn Reson Med** 2017 
+-  Smith et al. `Lateral corticospinal tract damage correlates with motor output in incomplete spinal cord injury. <http://www.sciencedirect.com/science/article/pii/S0003999317312844>`__ **Archives of Physical Medicine and Rehabilitation** 2017 
+-  Prados et al. `Spinal cord grey matter segmentation challenge. <https://www.sciencedirect.com/science/article/pii/S1053811917302185#f0005>`__  **Neuroimage** 2017
+-  Peterson et al. `Test-Retest and Interreader Reproducibility of Semiautomated Atlas-Based Analysis of Diffusion Tensor Imaging Data in Acute Cervical Spine Trauma in Adult Patients. <https://www.ncbi.nlm.nih.gov/pubmed/28818826>`__ **AJNR Am J Neuroradiol.** 2017 Oct;38(10):2015-2020 
+-  Kafali et al. `Phase-correcting non-local means filtering for diffusion-weighted imaging of the spinal cord. <http://onlinelibrary.wiley.com/doi/10.1002/mrm.27105/full>`__ **Magn Reson Med** 2018 
+-  Albrecht et al. `Neuroinflammation of the spinal cord and nerve roots in chronic radicular pain patients. <https://www.ncbi.nlm.nih.gov/pubmed/29419657>`__ **Pain.** 2018 May;159(5):968-977. doi: 10.1097/j.pain.0000000000001171
```

Comment:
This citation has a few extra details that the others don't. So, maybe we could remove them to keep the formatting consistent?

```suggestion
-  Albrecht et al. `Neuroinflammation of the spinal cord and nerve roots in chronic radicular pain patients. <https://www.ncbi.nlm.nih.gov/pubmed/29419657>`__ **Pain.** 2018
```